src/tasks/audio.c

```diff
@@ -52,8 +53,422 @@ struct audio_data {
 	struct pipeline *p;
 };
 
+#if defined(CONFIG_SUECREEK)
+
+/* Temporarily place the topology into a C array */
+
+unsigned char test_tone_playback_tplg[] = {
```

Comment:
This ptr will eventually come from linker script variable, but this is fine atm.